src/js/components/Data/DataForm.js

```diff
@@ -181,12 +181,18 @@ const resetPage = (nextFormValue, prevFormValue) => {
 };
 
 const transformTouched = (touched, value) => {
+  // DataFilters expects values for keys touched to evaluate to falsey to 
+  // not cause a badge. However any property value that is set back to its
+  // default/initial value that isn't undefined/null/false/0 will cause a
+  // badge this is particulary true for a 'range' which will always have
+  // a value.
+  //
+  // Should this instead determine touched by comparing against
+  // initial/default values?
```

Comment:
From a design standpoint (and to be handled in separate PR), but I might argue that if there are default values applied, the badge should appear. Rationale there being badge is indicating if filters have been applied, regardless of it that's due to default values/views or the user touching the filter directly.

Agree that in all of these cases, "range" is an outlier since it technically always has a value and we want a way not to badge it when the value is equal to min and max.

Reply:
I agree that if the default values are something other than the min/max then a badge should appear. I think it makes sense to just check against the min/max for range and if the values are not equal to that show a badge.